packages/contracts/docs/CHAINS.md

```diff
@@ -32,7 +32,7 @@ This document lists all the addresses of the contracts that have been deployed o
 
 | Contract              | Address                                    |
 |-----------------------|--------------------------------------------|
-| BulkVotingStrategy    | 0x9d55810da86A9610Ab3223f4CDF0b5d81FfA579c |
+| QuadraticFundingVotingStrategy    | 0x9d55810da86A9610Ab3223f4CDF0b5d81FfA579c |
```

Comment:
small nitpick but could you update the formatting of the table using something like https://www.tablesgenerator.com/markdown_tables